pom.xml

```diff
@@ -1,19 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>scalecube-organization-parent</artifactId>
-  <version>2.0.14-SNAPSHOT</version>
-
-
   <parent>
     <groupId>io.scalecube</groupId>
     <artifactId>scalecube-parent-pom</artifactId>
     <version>0.0.17</version>
   </parent>
+
+  <artifactId>scalecube-organization-parent</artifactId>
+  <version>2.0.14-SNAPSHOT</version>
+
   <packaging>pom</packaging>
 
-  <name>scalecube-organization-parent</name>
+  <name>ScaleCube Organization Service</name>
```

Comment:
this is not the service itself. please consider another name

Reply:
Just aligned with a similar name in `scalecube-configuration-service`

https://github.com/scalecube/scalecube-configuration-service/blob/9b03721dd77f92a90df831e65200f171f5ef7d5f/pom.xml#L15